tracer/src/Datadog.Trace/Span.cs

```diff
@@ -153,8 +150,9 @@ public override string ToString()
             sb.AppendLine($"OperationName: {OperationName}");
             sb.AppendLine($"Resource: {ResourceName}");
             sb.AppendLine($"Type: {Type}");
-            sb.AppendLine($"Start: {StartTime}");
+            sb.AppendLine($"Start: {StartTime.ToString("O")}");
             sb.AppendLine($"Duration: {Duration}");
+            sb.AppendLine($"End: {StartTime.Add(Duration).ToString("O")}");
```

Comment:
Arguably we don't need this I guess, as the time of the log is when it was written, but why not :D

Reply:
this has better precission than the log time.